---
title: "Single-cell sequencing of the blood T cell repertoire"
title-slide-attributes:
    data-background-color: black
    data-background-image: imgs/cover.png
    data-background-size: contain
    data-background-opacity: "0.2"
subtitle: "before and after trastuzumab treatment in early-stage HER2+ breast cancer"
date: March 14 2024
date-format: long
author: "John Vivian, Ph.D"
css: styles.css
format: 
  revealjs:
    controls: true
    transition: 'slide'
    center: true
    height: 1117
    width: 1728
editor: 
    renderOnSave: true
---

# Introduction  {background-color=#f0f4f7}

## {background-iframe="https://jvivian.github.io/"}

# Career Timeline {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="20%"}
::::
::::{.column width="60%"}
![](imgs/timeline.png)
::::
:::


## Nanopore Research {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="15%"}
::::
::::{.column width="85%"}
![](imgs/timeline_nanopore.png)
::::
:::


## Nanopore reread project {transition="zoom"}

![**A**: Substrate bound to the lipid bilayer via a cholesterol tag with a helicase (Hel308) bound and inactive at the other end. **B**: Electric potential causes the DNA duplex to unwind, leaving behind the cholesterol tether bound to the complement strand. **C**: Electric potential and the steady-state nature of the G-Quadruplex (GQ) causes it to unfold allowing the helicase to become enzymatically active in the 3’ -> 5’ direction. **D**: As Hel308 translocates the DNA back up through the pore, the GQ will refold allowing an additional enzyme to become bound. **E**: Continued translocation. **F**: More than 3-4 abasic residues causes Hel308 to dissociate from the substrate (9 are used in the substrate). The DNA strand is then pulled back down and the cycle is able to repeat itself once more.](imgs/nanopore_figure.png){width=100%}

## Computational Genomics Lab {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="20%"}
::::
::::{.column width="80%"}
![](imgs/timeline_grad.png)
::::
:::

<!-- ## Computational Genomics Lab {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="50%"}
![](imgs/timeline_grad.png)
::::
::::{.column width="50%"}
![](imgs/papers.png)
::::
::: -->


<!-- ## {background-image=imgs/toil_splash.png} -->
## {background-iframe="https://toil.ucsc-cgl.org/"} 

## {background-image="imgs/toil-splash-page.png" background-opacity=0.25 transition="convex"auto-animate=true}

:::{.columns}
::::{.column width="25%"}
::::
::::{.column width="50%"}
![](imgs/toil.png)
::::
:::


## Data Access: GTEx and UCSC Genome Browser{background-image="imgs/toil-splash-page.png" background-opacity=0.1 transition="convex" auto-animate=true}
:::{.columns}
::::{.column width="55%"}
![**UCSC Genome Browser**: A detailed 6700 bp exon-focused snapshot of a 43 Kbp segment on human chromosome 17 from the UCSC Genome Browser and highlights the tissue-specific expression of two genes as identified by GTEx RNA-seq. Tracks show tissue-specific expression of two genes from the GTEx dataset. 33 samples and 5 tissues displayed out of 7,304 samples and 53 tissues. The tracks for hg19 were produced using the UCSC liftOver tool, which converts coordinates from hg38 bedGraph files created by STAR2 within the Toil pipeline.](imgs/gtex.png)
::::
::::{.column width="45%"}
![](imgs/UCSC-Genome-browser-paper.png)<br>
::::
:::

## Data Access: GTEx and UCSC Genome Browser{background-image="imgs/toil-splash-page.png" background-opacity=0.1 transition="convex" auto-animate=true}
:::{.columns}
::::{.column width="50%"}
![**UCSC Genome Browser**: A detailed 6700 bp exon-focused snapshot of a 43 Kbp segment on human chromosome 17 from the UCSC Genome Browser and highlights the tissue-specific expression of two genes as identified by GTEx RNA-seq. Tracks show tissue-specific expression of two genes from the GTEx dataset. 33 samples and 5 tissues displayed out of 7,304 samples and 53 tissues. The tracks for hg19 were produced using the UCSC liftOver tool, which converts coordinates from hg38 bedGraph files created by STAR2 within the Toil pipeline.](imgs/gtex.png)
::::
::::{.column width="50%"}
![](imgs/sc_gtex_paper.png)<br>
::::
:::

## {background-iframe="https://xena.ucsc.edu"}

## {background-image="imgs/ucsc-xena-splash.png" background-opacity=0.2 transition="zoom"}
:::{.columns}
::::{.column width="10%"}
::::
::::{.column width="50%"}
![**UCSC Xena Browser**: FOXM1 and its isoforms across 19,948 **TCGA, TARGET, GTEx** samples, noting upregulation of FOXM1B and C in tumors versus GTEx normals using UCSC Xena Browser](imgs/xena.png)
::::
:::


:::{.notes}
- Collaborated with researchers in the ucsc genome browser team to create RNA-seq expression browser tracks
- Developed the toil-RNAseq pipeline
- TCAP, known as the titin cap protein, is predominantly expressed in muscle tissue. On the other hand, PP1R1B, which is a potential therapeutic target for neurological disorders, is expressed in the brain's basal ganglia but not in muscle or brain cortex
- Collaborated with the UCSC Xena team to provide an open-source data exploration platform of all ~20,000 samples
:::


## Research Data Collaborations {transition='concave'}
![](imgs/UCSC-treehouse-splash.png)

## {transition="convex" background-image="imgs/UCSC-treehouse-splash.png" background-opacity=0.3 auto-animate="true"}
:::{.columns}
::::{.column width="20%"}
::::
::::{.column width="80%"}
![](imgs/jco_paper_title.png)
::::
:::

## {transition="convex" background-image="imgs/UCSC-treehouse-splash.png" background-opacity=0.3 auto-animate="true"}
:::{.columns}
::::{.column width="10%"}
::::
::::{.column width="50%"}
![](imgs/jco_paper_title.png)
::::
::::{.column width="40%"}
![The model represented in Bayesian plate notation. **G** and **D** stand for Gene and Dataset respectively. $x$ denotes the gene expression from a background dataset, forming a convex combination $\hat{y}$ when multiplied by the dot product of $\beta^T$. The model incorporates a 'jointly Dirichlet' distribution for $\beta$, as each dataset contributes to the overall distribution. We add a Laplacian error term $\epsilon$ to the expected expression $\hat{y}$ when modeling the observed expression of a new sample $y$.](imgs/jco_plate_notation.jpeg)
::::
:::

## Atreca Inc. {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="25%"}
::::
::::{.column width="75%"}
![](imgs/timeline_atreca.png)
::::
:::

::: {.notes}
- Mention collaborating closely with protein engineering to design, maintain, improve tools they used every day
- Ab-CRUNCH
    - Annotated reports of antibody sequences, metadata, chain sequences, multiple alignments, and more
    - 200 antibodies took 45 - 60 minutes to process, after I rewrote it from scratch I reduced the codebase by 66% and 200 antibodies could be processed and returned in about a minute.
:::

## Novel Antibody Target Discovery Platform {auto-animate="true" background-color=#f0f4f7}
:::{.columns}
::::{.column width="60%"}
![](imgs/atreca_paper.png)
::::
::::{.column width="40%"}
![](imgs/timeline_atreca.png)
::::
:::

## Teiko Bio {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="15%"}
::::
::::{.column width="85%"}
![](imgs/timeline_teiko.png)
::::
:::


## Next Steps? {background-color=#f0f4f7 auto-animate="true"}
:::{.columns}
::::{.column width="5%"}
::::
::::{.column width="75%"}
![](imgs/timeline_ideaya.png)
::::
:::

# Analysis of T-cell sub-populations {background-color=#f0f4f7}

## Study Background {background-color=#f0f4f7}
:::{.columns}
::::{.column width="50%"}
:::{.center}
**HER2+ Breast Cancer**
::: 
::: {.incremental}
- HER2-positive breast cancer accounts for approximately 20% of breast cancer cases and is characterized by the overexpression of the human epidermal growth factor receptor 2 (HER2) gene [^1]
- HER2+ breast cancer is aggressive and accompanies poorer prognosis compared to HER2-negative breast cancers
- Better understanding of the biomolecular underpinnings of HER2+ breast cancer have paved the way for targeted therapies, significantly improving patient outcomes
::: 
::::
::::{.column width="50%"}
:::{.incremental}
:::{.center}
**Anti-HER2 Treatment**
::: 
- Anti-HER2 treatments, such as trastuzumab, have drastically the management of HER2-positive breast cancer, turning a once difficult-to-treat cancer into one with much more favorable outcomes [^2]
- These treatments specifically target the HER2 protein, inhibiting cancer cell growth and survival
- Ongoing research is focused on overcoming resistance to anti-HER2 therapies and enhancing treatment efficacy through combination therapies and novel agents
:::
::::
:::


[^1]: Slamon, D. J., et al. (2001). "Use of chemotherapy plus a monoclonal antibody against HER2 for metastatic breast cancer that overexpresses HER2." New England Journal of Medicine, 344(11), 783-792.

[^2]: Piccart-Gebhart, M. J., et al. (2005). "Trastuzumab after adjuvant chemotherapy in HER2-positive breast cancer." New England Journal of Medicine, 353(16), 1659-1672.

## Study Design 
:::{.columns}
::::{.column width="15%"}
::::
::::{.column width="85%"}
![](imgs/sc_fig1.png)
::::
:::


## Methods and Findings {background-color=#f0f4f7}
- Single-cell sequencing approach
- Analyzing T cell profiles before and after treatment
- Key findings and their significance

## Implications and Future Directions {background-color=#f0f4f7}
- Impact on understanding immune responses
- Potential influence on treatment strategies
- Future research directions

# Summary {background-color=#f0f4f7}

## Summary {background-color=#f0f4f7}
- Recap of your background and research
- How it aligns with the position

## Multiple Avenues for Contribution {background-color=#f0f4f7}

![](imgs/skills.png)

# Open for Questions {background-color=#f0f4f7}

## {background-iframe="https://jvivian.github.io/blog/posts/dr-dfm-covid/presentation-post.html"}

Thank you for your time and consideration